do.sh

```diff
@@ -152,7 +152,7 @@ e2e_build_jsmodule() {
 }
 
 e2e_build_library() {
```

Comment:
Can you rename the e2e_build_library to e2e_build_library_ctx1 and have the new e2e_build_library call that instead? I wanted to switch the defaults at some point.